### PR TITLE
feat: extract 24 h ticket out of periodic ticket 2

### PR DESCRIPTION
**Pull request by Nisha1306**
1. It has been decided that user should be able to buy 24h ticket anonymously unlike other periodic tickets. 

2. This PR also requires changes in Fire store to update preassignedFareProducts_v2.

- change ticket type of 24 hour ticker to 'hour24' from 'period'
- add isApplicableOnSingleZoneOnly as true for 24h ticket
- remove Student as a valid user for the same

 The changes have been done in  firestore config in staging but should be done in PROD file while moving this change to production. 

Github issue: https://github.com/AtB-AS/kundevendt/issues/56

### CODE REVIEW

**Review by gorandalum — CHANGES_REQUESTED** [review]
Good job. Some changes were suggested.

Also talk with Hilde/Simen if there should be any info to the user regarding that the product is only applicable for one zone. As of now there is no such information, and some users maybe won't understand why they only can select one zone.

File: src/translations/screens/Tickets.ts [hunk 1]
```diff
@@ -99,6 +99,15 @@ const TicketsTexts = {
       pilot_description: _('', ''),
       transportModes: _('Buss/trikk', 'Bus/tram'),
     },
+    hour24: {
+      title: _('24-timersbillett', '24 hour pass'),
+      description: _(
+        'Når du vil reise flere ganger på et døgn',
+        'For travelling several times in 24 hours',
```
[the remark this answers is not in the document]
Comment (Nisha1306) [reply src/translations/screens/Tickets.ts:106]: mmm referred Figma for this, but I can surely cross check with Hilde.

File: src/screens/Ticketing/Purchase/Overview/index.tsx [hunk 2]
```diff
@@ -166,6 +166,9 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
             fromTariffZone={fromTariffZone}
             toTariffZone={toTariffZone}
             style={styles.selectionComponent}
```
[the remark this answers is not in the document]
Comment (gorandalum) [reply src/screens/Ticketing/Purchase/Overview/index.tsx:168]: Great!

File: src/translations/screens/Tickets.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (gorandalum) [reply src/translations/screens/Tickets.ts:106]: Aha. Didn't know the english texts were in Figma. Often there is only norwegian texts in Figma, and us developers make the transalation on the fly. If in Figma this text is probably ok :+1:

File: src/screens/Ticketing/Purchase/TariffZones/index.tsx [hunk 3]
```diff
@@ -389,31 +404,33 @@ const TariffZones: React.FC<Props> = ({navigation, route: {params}}) => {
             highlighted={selectedZones.selectNext === 'from'}
             testID="searchFromButton"
           />
-          <ButtonInput
```
[the remark this answers is not in the document]
Comment (Nisha1306) [reply src/screens/Ticketing/Purchase/TariffZones/index.tsx:392]: fixed and pushed new commit.

**Review by gorandalum — APPROVED** [review]
Looks good! :+1: Note that I haven't tested accessibility on this.

**Comment by Nisha1306** [comment]
> Looks good! 👍 Note that I haven't tested accessibility on this.

I tested it a bit and I think it's fine.

**Pull request merged**